tests/platform_tests/api/test_fan.py

```diff
@@ -82,6 +130,7 @@ def test_get_model(self, duthost, localhost, platform_api_conn):
 
             if self.expect(model is not None, "Unable to retrieve fan {} model".format(i)):
                 self.expect(isinstance(model, STRING_TYPE), "Fan {} model appears incorrect".format(i))
+                self.compare_value_with_device_facts('model', model)
```

Comment:
`model` has the potential to vary by device, so we are not expecting this value to be stored in the platform.json file. We should not perform a comparison here.

Reply:
removed

Reply:
I misread that you were gathering this value from the device facts, not the platform facts. However, we have not yet defined a schema for storing these values in the inventory file. This can be added back at a later time if we decide to store the values there.